src/main/java/org/folio/service/transactions/batch/BatchTransactionChecks.java

```diff
@@ -263,4 +266,18 @@ private static void checkRestrictedEncumbrance(Budget budget, String currency, B
       throw new HttpException(422, error);
     }
   }
+
+  private static void checkPaymentsAndCreditsAmounts(Batch batch) {
+    List<Transaction> newPaymentsAndCredits = batch.getTransactionsToCreate().stream()
+      .filter(tr -> List.of(PAYMENT, CREDIT).contains(tr.getTransactionType()))
+      .toList();
+    for (Transaction tr : newPaymentsAndCredits) {
+      if (tr.getAmount() < 0) {
+        List<Parameter> parameters = new ArrayList<>();
```

Comment:
**Minor**
`List<Parameter> parameters = List.of(...);`

Reply:
changed